docs/howto/writing-migrations.txt

```diff
@@ -175,8 +177,11 @@ the respective field according to your needs.
 * Now you can apply the migrations as usual with the :djadmin:`migrate` command.
 
   Note there is a race condition if you allow objects to be created while this
-  migration is running. Objects created after the ``AddField`` and before
-  ``RunPython`` will have their original ``uuid``’s overwritten.
+  migration is running. The filter ``uuid__isnull=True`` helps to not overwrite
+  the original ``uuid`` of objects created after the ``AddField`` and before
+  ``RunPython``. If an object with a null field is created after ``RunPython``
+  before ``AlterField`` it can be fixed by reverting ``RunPython`` and running
+  it again that is fast and it only updates the new object.
```

Comment:
This last sentence could be parenthesised. The phrasing 

    reverting ``RunPython``

should be replaced the specific `migrate` step.

Reply:
Sorry I don't follow your suggestion, how do you suggest it be rephrased?